SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -66,43 +66,55 @@ void USpatialSender::Init(USpatialNetDriver* InNetDriver)
 	ClassInfoManager = InNetDriver->ClassInfoManager;
 }
 
+const WorkerAttributeSet ServerWorkerAttributeSet = { SpatialConstants::ServerWorkerType };
```

Comment:
Isn't this already defined in SpatialConstants.h? If so I would use those instead:)